db/blip_collection_context.go

```diff
@@ -34,6 +34,10 @@ type blipSyncCollectionContext struct {
 	sgr2PushAlreadyKnownSeqsCallback func(alreadyKnownSeqs ...SequenceID)           // sgr2PushAlreadyKnownSeqsCallback is called to mark the sequence as being immediately processed
 	emptyChangesMessageCallback      func()                                         // emptyChangesMessageCallback is called when an empty changes message is received
 
+	// if we encounter a situation where a requested revision is not available, should we try sending an alternative as a replacement revision?
```

Comment:
```suggestion
	// when a requested revision is not available, whether we should send the active revision as a replacement revision (depending on access)
```
If this change to the comment is accurate, I think it's more descriptive.